### DISCUSSION TITLE
UAC without installation

### DISCUSSION THREAD

**Discussion by cfr2305** [rustdesk/rustdesk#859] [General]
Hi,
I found this reddit post and have the same issue:
https://www.reddit.com/r/rustdesk/comments/vh987e/something_like_quicksupport/

Is it planed to build in a way to run RustDesk as administrator on the remote side (without giving the user the administrator password) to solve the UAC issue? (like Teamviewer or Anydesk does it?)

Thanks.

**Comment by rustdesk** [comment]
Long term, yes.

**Reply by wneiner** [reply]
Is there an estimate of long term?  Any ideas of what can be done to enable it?

Any insight is most appreciated.

Thank you,

**Reply by neogalaxy88** [reply]
some update ?
Thank you

**Comment by rustdesk** [comment]
#70

**Comment by BloodyIron** [comment]
Entering a remote login, even elevated, is not the same as interacting with UAC.
